[Jul 23 18:02:08] INFO[29592]: asterisk.test_case:173 __init__: Executing tests/apps/bridge/bridge_add
[Jul 23 18:02:08] INFO[29592]: asterisk.test_case:278 create_asterisk: Creating Asterisk instance 1
[Jul 23 18:02:09] INFO[29592]: asterisk.test_case:444 _start_asterisk: Starting Asterisk instance 1
[Jul 23 18:02:11] INFO[29592]: asterisk.test_case:329 create_ami_factory: Creating AMIFactory 1 to 127.0.0.1
[Jul 23 18:02:11] INFO[29592]: twisted:154 publishToNewObserver: Starting factory <starpy.manager.AMIFactory instance at 0x7fba00e6def0>
[Jul 23 18:02:11] INFO[29592]: AMI:184 connectionMade: Connection Made
[Jul 23 18:02:11] INFO[29592]: AMI:203 onComplete: Login Complete: {u'message': u'Authentication accepted', u'response': u'Success', u'actionid': u'35cc2ba0add2-140436855775032-1'}
[Jul 23 18:02:11] INFO[29592]: asterisk.test_case:582 _ami_connect: AMI Connect instance 1
[Jul 23 18:02:26] WARNING[29592]: asterisk.test_case:529 _reactor_timeout: Reactor timeout: '15' seconds
[Jul 23 18:02:26] INFO[29592]: asterisk.test_case:487 __stop_instances: Stopping Asterisk instance 1
[Jul 23 18:03:11] WARNING[29592]: asterisk.asterisk:580 __send_kill: Sending KILL to Asterisk 127.0.0.1
[Jul 23 18:03:11] WARNING[29592]: asterisk.ami:208 __check_result: Event occurred 0 times, which is out of the allowable range
[Jul 23 18:03:11] WARNING[29592]: asterisk.ami:209 __check_result: Event description: {'count': 1, 'conditions': {'match': {'Variable': 'AMDSTATUS', 'Value': 'HUMAN', 'Event': 'VarSet', 'Channel': 'Local/bob@default-00000001;2'}}}
[Jul 23 18:03:11] WARNING[29592]: asterisk.ami:208 __check_result: Event occurred 0 times, which is out of the allowable range
[Jul 23 18:03:11] WARNING[29592]: asterisk.ami:209 __check_result: Event description: {'count': 1, 'conditions': {'match': {'Variable': 'AMDSTATUS', 'Value': 'HUMAN', 'Event': 'VarSet', 'Channel': 'Local/charlie@default-00000002;1'}}}
[Jul 23 18:03:11] INFO[29592]: asterisk.test_case:508 __stop_reactor: Stopping Reactor
[Jul 23 18:03:11] INFO[29592]: AMI:1136 clientConnectionLost: connection lost, reconnecting...
[Jul 23 18:03:11] INFO[29592]: twisted:154 publishToNewObserver: <twisted.internet.tcp.Connector instance at 0x7fba00d2b098> will retry in 2 seconds
[Jul 23 18:03:11] INFO[29592]: AMI:1139 clientConnectionLost: None
[Jul 23 18:03:11] INFO[29592]: twisted:154 publishToNewObserver: Stopping factory <starpy.manager.AMIFactory instance at 0x7fba00e6def0>
[Jul 23 18:03:11] INFO[29592]: twisted:154 publishToNewObserver: Main loop terminated.
[Jul 23 18:03:11] INFO[29592]: test_runner:308 main: Test run for tests/apps/bridge/bridge_add completed with result False
